src/common/config_opts.h

```diff
@@ -1305,7 +1306,7 @@ OPTION(filestore_op_thread_timeout, OPT_INT, 60)
 OPTION(filestore_op_thread_suicide_timeout, OPT_INT, 180)
 OPTION(filestore_commit_timeout, OPT_FLOAT, 600)
 OPTION(filestore_fiemap_threshold, OPT_INT, 4096)
-OPTION(filestore_merge_threshold, OPT_INT, 10)
+OPTION(filestore_merge_threshold, OPT_INT, -1)
```

Comment:
needs to be -10 to keep the same settings - the split threshold is 16 * abs(merge_threshold) * split_mulitple